.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea
```

Comment:
update the gitignore to not include bin and other properties files.

Reply:
will fix it